keps/sig-node/2371-cri-pod-container-stats/README.md

```diff
@@ -540,19 +539,19 @@ As a requirement for the Beta stage, cAdvisor must support optionally collecting
 
 - CRI should be extended to provide required stats for `/stats/summary`
 - Kubelet should be extended to provide the required stats from CRI implementation for `/stats/summary`.
-- cAdvisor should be updated to support no longer collecting stats that are duplicated with CRI implementation.
 - This new behavior will be gated by a feature gate to prevent regressions for users that rely on the old behavior.
-- Conduct research to find the set of metrics from `/metrics/cadvisor` that compliant CRI implementations must expose.
-- Conformance tests for the fields in `/metrics/cadvisor` should be created
-- Performance tests for CPU/memory usage between Kubelet/cAdvisor/CRI implementation should be added.
+
 #### Alpha -> Beta Graduation
 
-- CRI implementations should report any difficulties collecting the stats, and by Beta the CRI stats implementation should perform better than they did with CRI+cAdvisor.
-- CRI implementations should support their equivalent of `/metrics/cadvisor`, passing the performance and conformance suite created in Alpha.
+- Conduct research to find the set of metrics from `/metrics/cadvisor` that compliant CRI implementations must expose.
+- Conformance tests for the fields in `/metrics/cadvisor` should be created.
```

Comment:
Full k8s conformance tests will be significant effort, perhaps instead we should just ensure that the fields we want to support have overlap?

Reply:
how will we continually verify that without testing? especially as time goes on. If we consider these fields a sort of  API of kube, then we should probably do something to verify they're there